packages/medusa-events-typeorm/src/loaders/index.ts

```diff
@@ -0,0 +1,20 @@
+import glob from 'glob';
+import { Logger, MedusaContainer } from '@medusajs/medusa/dist/types/global';
+
+export default async function authStrategiesLoader(container: MedusaContainer) {
+	const logger = container.resolve<Logger>('logger');
+	try {
+		const files = glob.sync('../event-subscribers/[!__]*.js', {});
```

Comment:
Suggestion : should we just ask for an index file from where they are all exported? Also, if you use path.resolve and use import it should work